### ISSUE TITLE
Add optional tools to Airy

### ISSUE THREAD

**Issue by ljupcovangelski** [airyhq/airy#4020]
Add helm charts to deploy `redis` and `postgresql` servers inside the Airy instance.

**Comment by AitorAlgorta** [comment]
this is done right? @ljupcovangelski

**Comment by ljupcovangelski** [comment]
@AitorAlgorta yes, but I'd keep it open until we don't fix the display on the `status page`.
